Corona-Warn-App/src/main/res/values-de/dcc_reissuance_strings.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+
+    <!-- DCC Reissuance: Success Screen -->
+
+    <!-- XHED: DCC reissuance success screen title -->
+    <string name="dcc_reissuance_success_screen_title">Zertifikat aktualisiert</string>
+
+    <!-- YTXT: Headline for DCC reissuance success screen -->
+    <string name="dcc_reissuance_success_screen_headline">Ihr Zertifikat wurde erfolgreich aktualisiert</string>
+
+    <!-- YTXT: Body for DCC reissuance success screen -->
+    <string name="dcc_reissuance_success_screen_body">Sie finden das aktualisierte Zertifikat nun im Zertifikate Tab.</string>
```

Comment:
Zertifikate-Tab

Reply:
@KathrinLuecke 
CV113 is suggesting Zertifikate-Tab, you are suggesting Zertifikate Tab.. whats next?

Reply:
I am suggesting "Zertifikate-Registerkarte". We used "Registerkarte" also in other places, not "Tab".

Reply:
yes but @CV113  is writing Tab ?
@KathrinLuecke it is your suggestion now. please approve